pkg/cloud/gcp/actuators/machineset/controller.go

```diff
@@ -156,6 +157,19 @@ func (r *Reconciler) reconcile(machineSet *machinev1.MachineSet) (ctrl.Result, e
 	machineSet.Annotations[cpuKey] = strconv.FormatInt(machineType.GuestCpus, 10)
 	machineSet.Annotations[memoryKey] = strconv.FormatInt(machineType.MemoryMb, 10)
 
+	if (len(providerConfig.GuestAccelerators)) == 0 {
```

Comment:
Nit, no need for extra parenthesis
```suggestion
	if len(providerConfig.GuestAccelerators) == 0 {
```